flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/stream/sql/WindowAggregateTest.scala

```diff
@@ -239,4 +252,179 @@ class WindowAggregateTest extends TableTestBase {
       )
     streamUtil.verifySql(sql, expected)
   }
+  
+   @Test
+  def testMaxAggregatation(): Unit = {
```

Comment:
I thought you refer to what i added in scala/stream/sql/SqlITCase.scala
...which is the partitioned and non-partitioned verification of 2 aggregates

..i am not sure what you mean with checking the plan....
@fhueske